src/EventListener/DeserializeListener.php

```diff
@@ -52,7 +52,10 @@ public function onKernelRequest(GetResponseEvent $event)
             || $request->isMethod('DELETE')
             || !($attributes = RequestAttributesExtractor::extractAttributes($request))
             || !$attributes['receive']
-            || ('' === ($requestContent = $request->getContent()) && $request->isMethod('PUT'))
+            || (
+                    '' === ($requestContent = $request->getContent())
+                    && in_array($request->getMethod(), [Request::METHOD_POST, Request::METHOD_PUT])
```

Comment:
Can you use two `=== ` instead please (it's faster).

Reply:
no problem.